tcs-service/src/main/resources/db/migration/schema/V4.18__adding_postEsrEvent.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE `PostEsrEvent` (
+  `id` bigint(20) NOT NULL AUTO_INCREMENT,
+  `postId` bigint(20) NOT NULL,
+  `eventDateTime` datetime NOT NULL,
+  `filename` varchar(255) NOT NULL,
+  `positionNumber` bigint(20) NOT NULL,
+  `positionId` bigint(20) NOT NULL,
+  `status` varchar(255) NOT NULL,
+  PRIMARY KEY (`id`),
+  KEY `PostEvnt_Post_id` (`postId`),
+  CONSTRAINT `PostEvnt_Post_id` FOREIGN KEY (`postId`) REFERENCES `Post` (`id`)
```

Comment:
The name of foreign keys _generally_ follow the pattern `fk_{TABLE_NAME}_{FIELD_NAME}`  where `{FIELD_NAME}` is changed to `post_id` like you have done.  I see that's not true of the `PlacementEsrEvent` table :see_no_evil: